frontend/src/tests/components/Forms/__snapshots__/MineRecordForm.spec.js.snap

```diff
@@ -202,26 +202,7 @@ ShallowWrapper {
         "PLR": "Placer",
       }
     }
-    mineTenureTypes={
-      Array [
-        Object {
-          "label": "Coal",
-          "value": "COL",
-        },
-        Object {
-          "label": "Mineral",
-          "value": "MIN",
-        },
-        Object {
-          "label": "Placer",
-          "value": "PLR",
-        },
-        Object {
-          "label": "BC Land",
-          "value": "BCL",
-        },
-      ]
-    }
+    mineTenureTypes={undefined}
```

Comment:
this doesn't look good. Probably want to check on this one.

Reply:
Fixed, as well as in a few other places.